pkgs/applications/misc/confy/default.nix

```diff
@@ -0,0 +1,58 @@
+{ appstream-glib
+, desktop-file-utils
+, fetchurl
+, gobject-introspection
+, gtk3
+, lib
+, libnotify
+, libhandy
+, meson
+, ninja
+, pkg-config
+, python3
+, stdenv
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "confy";
+  version = "0.6.4";
+
+  src = fetchurl {
+    url = "https://git.sr.ht/~fabrixxm/confy/archive/${version}.tar.gz";
+    sha256 = "0v74pdyihj7r9gb3k2rkvbphan27ajlvycscd8xzrnsv74lcmbpm";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib.dev
```

Comment:
You need appstream-builder or so? Why? We don't need to build that.

Reply:
Meson looks for the `appstream-util` program, but I removed this dependency and it still builds. Thanks